x-pack/plugins/security_solution/server/lib/detection_engine/rules/find_rules.ts

```diff
@@ -29,5 +29,6 @@ export const findRules = ({
       sortOrder,
       sortField,
     },
+    includeSnoozeData: true,
```

Comment:
Does it mean the snooze data is fetched from somewhere else, e.g. task saved objects? Is there any impact on performance if it's set to true?

Reply:
No, it is because snooze is internal so we do not want to show it in our public API yet

Reply:
Gotcha! 👍